autoscale/src/main/resources/logback.xml

```diff
@@ -64,6 +64,10 @@
         <level value="${PERISCOPE_SQL_LOG_LEVEL:-INFO}"/>
     </logger>
 
+    <logger name="org.springframework.orm.jpa">
```

Comment:
Why is this needed? What additional logs can come out of this?

Reply:
Will check this. I think the intent was to reduce jpa logging... i.e. AutoScale itself at DEBUG level, but JPA at INFO level.